src/plugins/release-monitor/index.js

```diff
@@ -261,12 +264,17 @@ async function prOpenHandler(context) {
     );
 
     const allCommits = await context.github.pullRequests.getCommits(context.issue());
+    const isSemverPatchPr = isMessageValidForPatchRelease(getCommitMessageForPR(allCommits, context.payload.pull_request));
     const statusFunc =
-        releaseIssue.length === 0 || isMessageValidForPatchRelease(getCommitMessageForPR(allCommits, context.payload.pull_request))
+        releaseIssue.length === 0 || isSemverPatchPr
             ? createSuccessPRStatus
             : createPendingPRStatus;
 
-    await statusFunc(context, pluckLatestCommitSha(allCommits));
+    await statusFunc({
+        context,
+        sha: pluckLatestCommitSha(allCommits),
+        description: isSemverPatchPr ? "This change is semver-patch" : undefined // eslint-disable-line no-undefined
```

Comment:
It might be better to put the other message here too, and have no default message in `statusFunc`.

```js
description: isSemverPatchPr ? "This change is semver-patch" : "No patch release is pending";
```

Reply:
That function get called from other places also n it uses the default message all the time except for this exception. That why I did it like this.